terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1244,6 +1245,15 @@ def remove_inconsequential_statements(statements: list[JSON]) -> list[JSON]:
                             - /var/run/docker.sock:/var/run/docker.sock
                           labels:
                             io.rancher.os.scope: "user"
+                        prune-images:
+                          image: {dind_image}
+                          volumes:
+                            - /var/run/docker.sock:/var/run/docker.sock
+                          command: exec gitlab-dind docker image prune --all --force
```

Comment:
```suggestion
                          command: exec gitlab-dind docker image prune --all --force --filter "until=30d"
```

Reply:
Don't delete images from more recent builds. If we deleted them we would risk failing the requirements check on sandbox builds since that check depends on image caching.